.gitlab-ci.yml

```diff
@@ -309,7 +309,7 @@ build-malus:
         --polkadot
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
-        $CARGO_UPDATE_CRATES
+        "$CARGO_UPDATE_CRATES"
```

Comment:
This looks right but it needs a change in https://github.com/paritytech/pipeline-scripts. I'll open a PR there and link it here.

Reply:
Why? In Substrate we already do this: https://github.com/paritytech/substrate/blob/c41c62ae4f43a930b247b5e7858a0eb3fd9147d4/.gitlab-ci.yml#L539

Reply:
Oh nevermind, I got confused by how shell expansion works.